### PR TITLE
revised activity_register.xml file

### PR DESCRIPTION
**Pull request by mirahazall**
## Description

Updated the register-activity.xml file to include a logo and revised the edit text fields. 
Created loading_page.xml file that includes a logo and a custom gradient progress bar.

### Todos

- [x] Tested and working locally
- [x] Code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] Code changes documented
- [x] Requested review from >= 2 devs on the team (one frontend and one backend recommended)

### How to test
Added front end components work as desired on emulator.

### Screenshots and/or Gifs
Screenshot 2024-09-03 at 10 55 05 PM
Screenshot 2024-09-03 at 10 53 27 PM

### Associated MS Planner Tasks
- [Splash and Register Screen](https://tasks.office.com/deakin365.onmicrosoft.com/Home/Task/U1oPlIjT60yGCb4M9uaH9MgALGYf?Type=TaskLink&Channel=Link&CreatedTime=638609649609530000)

### Known Issues
[Describe any known issue with this change]
No issues detected

### CODE REVIEW

File: app/src/main/res/layout/loading_page.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (HamzaTahir22) [inline app/src/main/res/layout/loading_page.xml:1]: Rename this file from loading_page.xml to splash_screen.xml

Comment (HamzaTahir22) [inline app/src/main/res/layout/loading_page.xml]: Rename this file from loading_page.xml to splash_screen.xml

**Review by HamzaTahir22 — APPROVED** [review]
Approving with minor comment. @mirahazall 
Please solve it and resubmit then merge,

**Review by AsherAli22 — APPROVED** [review]
LGTM

File: app/src/main/res/layout/loading_page.xml (hunk 1 above)
Comment (mirahazall) [reply app/src/main/res/layout/loading_page.xml]: I just did. Thank you!

**Pull request merged**